Source/Python/pyWarpX.cpp

```diff
@@ -55,6 +56,7 @@ PYBIND11_MODULE(PYWARPX_MODULE_NAME, m) {
     )pbdoc";
 
     // note: order from parent to child classes
+    init_WarpXParIter(m);
```

Comment:
Let's move this after we know the PC, one line down, because we have it as an argument to the pti constructor.